packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap

```diff
@@ -179,7 +179,7 @@ import rawModule from './raw.png'",
   "errors": [],
   "errorsCount": 0,
   "filteredModules": undefined,
-  "hash": "aa724e620c7dec1cbd69",
```

Comment:
hash change is caused by the name of SwcJsMinimizerPlugin ("javascript" -> "rspack.SwcJsMinimizerPlugin")